xml/System/ReadOnlySpan`1+Enumerator.xml

```diff
@@ -44,12 +60,20 @@
         </Attribute>
       </Attributes>
       <ReturnValue>
-        <ReturnType>System.T&amp;</ReturnType>
+        <ReturnType>T@</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a reference to the item at the current position of the enumerator.</summary>
```

Comment:
`<MemberSignature Language="C#" Value="public T&amp; Current { get; }" />`

[Remove the ampersand](https://github.com/dotnet/dotnet-api-docs/pull/358/files#diff-eb540cf08a14e0c666aab66049b91c77R46), like span:https://github.com/dotnet/dotnet-api-docs/blob/88663122a2ba4a93aa7d133c3423bd477688b928/xml/System/Span%601%2BEnumerator.xml#L56

Reply:
Is that normal to remove? I think all that header stuff was auto generated (I'm new to working on ref files)

Reply:
Good question, I am not sure.

The signature for Span is `public ref T Current` and for ReadOnlySpan is `public ref readonly T Current`

@weshaggard, any ideas?

Reply:
If the signature is wrong, we'd need to open an issue at https://github.com/mono/api-doc-tools/ 

/cc @joelmartinez

Reply:
I'm not sure what tool was used here to generate these but I would not be surprised at all if the tool didn't understand `ref` and `ref readonly` as they are pretty new concepts.

Reply:
@weshaggard is correct: the tool loses the `ref` and `readonly` modifiers. See   MicrosoftDocs/feedback#468

Reply:
what do we do about this issue?

Reply:
Sometimes, we've done workarounds to fix the docs manually here. The problem is that the next time we run CI, the workaround is gone. We can follow up with @dend and @joelmartinez to see when this issue will be picked up.

Reply:
Verified with @mairaw that this will be fixed by mdoc and not by hand editing.